packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -66,6 +66,9 @@ export async function genFinalS3ClientConfig(config: S3ClientConfig): Promise<Ba
         throw new Error(`S3 endpoint ${config.endpoint} cannot be https if sslEnabled is false`);
     }
 
+    config.responseChecksumValidation = 'WHEN_REQUIRED';
+    config.requestChecksumCalculation = 'WHEN_REQUIRED';
```

Comment:
Can you add a comment above these two lines that describes why you're adding these settings and link to the upstream issue that mentions these settings?

Reply:
Actually, you have a comment elsewhere in this PR, I guess this is sufficient.